lib/services/access_checker.go

```diff
@@ -857,7 +857,6 @@ func (a *accessChecker) HostUsers(s types.Server) (*HostUsersInfo, error) {
 		return strings.Compare(a.GetName(), b.GetName())
```

Comment:
IIRC we had to sort the roles to ensure some stable order of sudoer lines. Now that they're calculated in a different method, do we still need to sort the roles here, in `HostUsers()`?